bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -221,6 +222,11 @@ impl SlidingSyncRoom {
                     Err(RecvError::Closed) => break,
                     Ok(_) | Err(RecvError::Lagged(_)) => timeline.clear().await,
                 }
+
+                match ignore_user_list_rx.recv().await {
```

Comment:
Should this be put in a different loop than the `reset_broadcast_rx`? Aside from that, I wonder if breaking the loop is actually correct here. Once an error occurs it will never work properly again, no? 

Otherwise it feels like this is the correct place to reset the timeline, I wonder why it doesn't work properly.

Reply:
it works properly now! it was because we were using the same loop and breaking from the first receive check!